ts/src/base/Exchange.ts

```diff
@@ -3606,6 +3606,39 @@ export default class Exchange {
         }
     }
 
+    handleForTimeInForceFlags (isMarketOrder: boolean, targetFlagName: string, exchangeSpecificBoolean: boolean, params: any = {}) {
+        /**
+         * @ignore
+         * @method
+         * @param {string} type Order type
+         * @param {boolean} exchangeSpecificBoolean exchange specific postOnly
+         * @param {object} params exchange specific params
+         * @returns {[boolean, params]}
+         */
+        const timeInForce = this.safeStringUpper (params, 'timeInForce');
+        if (targetFlagName === 'postOnly') {
```

Comment:
why are we receiving this as a `parameter` and adding this if? `postOnly = true` is one of the unified ways to tell this is a post-only order we don't need to receive the `postOnly` key every time

Reply:
I have designed this method not only for `postOnly` but for any possible type (for future adoption) i.e. FOK, IOC or etc. that will avoid us having dozens of almost duplicate methods with similar bodies. 
so, inside implementation we can have:
```
[ postOnly, params ] = this.handleForTimeInForceFlags (type === 'market', 'postOnly', exchangeSpecificTimeInForce === 'poc', params);
[ is_ioc, params ]  = this.handleForTimeInForceFlags (type === 'market', 'ioc', exchangeSpecificTimeInForce === 'iOC', params);
[ is_fok, params ]  =  ...
```
so, that method will handle all internal checkings for possible issues and will shift all hardwork from implementation into base method. 
A few months ago, we didnt have `postOnly` as unified, but today we have. similar might come for FOK, IOC and others, and we should have a base method for them to be ready for adoption. so, passing `postOnly` unified keyname into that method is not bad imo.

Reply:
@ttodua That would be very confusing, first one method should do one thing and one thing only second just looking at the name you should be able to understand clearly what the method does so I don't think we should be mixing everything inside the same method.